src/constants.js

```diff
@@ -21,7 +21,7 @@ const ProdConstants = {
 };
```

Comment:
These constants are no longer required. Please remove them as these are now managed in a table

Reply:
Done, removed `TMOConstants`